### ISSUE TITLE
Reuse the same packet device across different run

### ISSUE THREAD

**Issue by frouioui** [vitessio/arewefastyet#26]
We are currently creating a new packet device for each individual run. Creating a new packet device takes up a lot of time, monetary resources, etc.

When in development, the time taken to create a new device is significantly increasing lead time. In fact, it takes about 10 minutes to spin up an OLTP test, where the test itself runs for less than a minute and the creation of the device takes approximately 9 minutes.

Instead, we can reuse the same device for a batch of runs/tasks. The Ansible codebase already integrates this feature. Current Ansible playbooks and roles allow us to "clean" the previous state. Among them, the following snippet enables us to stop previous services and remove all previously used Vitess data: 

https://github.com/vitessio/arewefastyet/blob/4f6626497f46cf7128ec06578444caf714af7c68/ansible/roles/vitess_build/tasks/clean.yml#L1-L25

Moreover, the `full.yml` file, which is the "main" playbook we use, is "customized" as followed:
https://github.com/vitessio/arewefastyet/blob/4f6626497f46cf7128ec06578444caf714af7c68/ansible/full.yml#L2-L3

When running ansible-playbook as shown in the following snippet, we can omit `provision` as explained in [this doc](https://github.com/vitessio/arewefastyet/tree/master/ansible#run-the-scripts), that way we can use the previous provisioned device by using the built Ansible file we used in `./ansible/build/`.

https://github.com/vitessio/arewefastyet/blob/6aea9b42a4ecd87660beff2c670f65d93b86b93e/run-oltp#L15

In order to integrate this with the current run-benchmark, changes are required in how we execute ansible-playbook.

**Comment by Akilan1999** [comment]
Good catch. I would give it a higher priority as you mentioned we should not be wasting resources.

**Comment by frouioui** [comment]
We can probably schedule this for after the workflows and molecule implementation (#25).

**Comment by Akilan1999** [comment]
Sure I agree

**Comment by frouioui** [comment]
Hello @vielmetti, thank you for pointing that out. I have indeed observed a strong increase in the number of instances, this problem can be temporarily fixed by #131. Regarding this improvement, I will move it up the priority list. In the meantime, I will carefully keep an eye on our resource usage.

**Comment by vielmetti** [comment]
Looks good @frouioui - I noticed a big drop in resource usage already.

**Comment by frouioui** [comment]
Hello @vielmetti!

Reusing devices across multiple executions will not be supported for now. From now on, we will run three concurrent executions per day at around the same time. Support for webhook-triggered executions is stopped since #163, and is replaced by corn jobs, those jobs will spin up three devices for more or less than an hour.

Closing this issue, excessive resource usage shouldn't be a problem anymore. Feel free to report any issue related to excessive consumption here.

Apologies for our extensive usage of your resources yesterday (Monday, May 10, 2021), tests were being performed to set up those cron jobs.
